### ISSUE TITLE
[BUG] - magenta line POL5R SID

### ISSUE THREAD

**Issue by omega892** [invernyx/md11-bugs#1037]
### Describe the bug

Having programmed the POL 5R from EGCC rwy 23R and also on another flight, the POL 1Y from EGCC rwy 23L, the SID navdata is incorrectly displayed on the ND and NAV mode will not follow the SID correctly. Note also that at XUMAT, there is a turn to the left to loop back to XUMAT which is incorrect

### Steps to reproduce

Programme the POL 5R from rwy 23R at EGCC and then attempt to fly it. Fly the POL 1Y from EGCC.

### Expected behaviour

The SID to be correctly displayed and followed by NAV / PROF.

### Evidence

![20240409170016_1](https://github.com/invernyx/md11-bugs/assets/152972533/79c861f1-4bf6-4383-9aee-4c72507c9b2f)

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Crash

No

### Phase of flight

Takeoff

### Altitude

Ground to FL150

### Departure

EGCC

### Arrival

EGCC

### Cruise level

FL150

### ZFW

N/A

### CG

N/A

### Fuel

N/A

### Cost index

N/A

### Route

POL 5R / POL 1Y SIDs

### Location

EGCC runway 23R and 23L

### Agreement

- [X] Yes

**Comment by omega892** [comment]
![20240411102924_1](https://github.com/invernyx/md11-bugs/assets/152972533/40ce25ac-3287-4d07-ae35-8daa86f8eb57)
Just flown the POL 1Y departure and even though the ND shows the SID drawn in a more understandable way, the ground track extends beyond the 3.2nm point on departure and overshoots the required groundtrack

**Comment by omega892** [comment]
![20240410143004_1](https://github.com/invernyx/md11-bugs/assets/152972533/63900b30-2a3b-41ea-9c8e-3a38425d1416)
![20240410143256_1](https://github.com/invernyx/md11-bugs/assets/152972533/2de84fad-2885-44e8-bd9d-e4ec4aa93b8a)
Couple more screenshots from the first example described.

**Comment by omega892** [comment]
![20240411103032_1](https://github.com/invernyx/md11-bugs/assets/152972533/762bb266-fe39-44f0-b24e-ab16b02eca35)

**Comment by joon44** [comment]
![Image](https://github.com/user-attachments/assets/7209ca2c-a3bf-44e5-9668-b42eca3486ea)

![Image](https://github.com/user-attachments/assets/d5ae5ce0-ad82-4423-a604-db1dd889371a)

**Comment by omega892** [comment]
So still not looking correct

**Comment by joon44** [comment]
> So still not looking correct

Which part is wrong? To me it looks exactly as shown on the chart.

**Comment by omega892** [comment]
So the turn after takeoff should be at 3nm from the runway according to the chart (D3.0 MCT) which makes it an overfly waypoint. From the ND screenshot you can see the turn is graphically represented showing an early turn prior to the D3.0 MCT (shown as MCT3 on the ND). So that to me doesn’t match. Also due to the early turn the 343 track isn’t intercepted until the D294G on the ND picture so the aircraft is effectively cutting the corner of the SID.

**Comment by joon44** [comment]
Yeah in fact you're right. Some overfly and flyby waypoints are treated incorrectly. MCT3 is treated as flyby I presume. I wonder if its a navdata issue or a plane issue. I was looking at your original report and what its drawing now

**Comment by omega892** [comment]
Yep it has improved, but it still isn’t correct and I don’t think it’s a Navdata issue. I was flying the DETLING 2G out of EGLL from 27L and way overshot the initial turn but will go back and have a look at that one again as I didn’t submit a bug report. Also, I haven’t checked to see if it is still misreading the navdata and drawing Missed Approach Points as ghost runways on the ND which no other aircraft I know does in MSFS.

**Comment by omega892** [comment]
Have just had a quick look at the latest update - first screenshot is the DET2G off 27L at EGLL showing a massive overshoot of the first turn. Second screenshot is showing double runway at Kai Tak with the "ghost" runway representing the Missed Approach Point so I think the FMC is misreading the data somehow. These "ghost" runways also appear at other airports with similar approaches.
![Image](https://github.com/user-attachments/assets/77d3164e-cc4b-4272-9763-14f29feabbf7)
![Image](https://github.com/user-attachments/assets/9f49d26d-ee16-49d0-a73e-6ef26bd8d117)

**Comment by turbofandude** [comment]
Should be resolved in the next update.
